pages/common/bitcoin-cli.md

```diff
@@ -0,0 +1,25 @@
+# bitcoin-cli
+
+> Command-line client to interact with the Bitcoin daemon via RPC calls.
+> The client uses the configurations defined in the `bitcoin.conf` file.
```

Comment:
Do we need to mention `The client` again here? How about this?

```suggestion
> Uses the configuration defined `bitcoin.conf`.
```

I might have the grammar slightly wrong here, as I'm unsure of the context surrounding `bitcoin.conf`.

Reply:
Sounds good :) I'll make the change.